### DISCUSSION TITLE
Support for WebAuthn and SvelteKit?

### DISCUSSION THREAD

**Discussion by ioucyf** [clerk/javascript#1099] [Support request - DO NOT OPEN HERE] [answered]
- Does clerk support webAuthn? 
- If not, Is it planned? 
- Is it possible to create an implementation of it for my web app using the existing API?

Different topic:
- Does clerk plan to officially support svelte/kit?
- If yes, how soon?

Thank you so much for your time. 🙏

**Comment by jescalan** [comment, marked as answer]
Sorry for the super late response here!

- Clerk does not support wbeauthn, and it is not currently on the roadmap, but it is something we are discussing support for down the road! I don't think it would be possible to create yourself, but you're welcome to give it a shot!
- We also do not have official support for svelte, but we have had a lot of requests for it and we'd like to add it as soon as we're able to!

EDIT: We are now [actively working on adding webauthn/passkeys support](https://feedback.clerk.com/roadmap/b596708b-08cf-46e3-96e7-16a32c0d2c32). Feel free to upvote or leave feedback here if you're interested in beta testing!

**Comment by LekoArts** [comment]
Update: Support for Passkeys is currently "In Progress" 🎉 
https://feedback.clerk.com/roadmap/b596708b-08cf-46e3-96e7-16a32c0d2c32
